packages/gestalt/src/ButtonSocial.tsx

```diff
@@ -0,0 +1,325 @@
+import { forwardRef, useImperativeHandle, useRef } from 'react';
+import getAriaLabel from './accessibility/getAriaLabel';
+import NewTabAccessibilityLabel from './accessibility/NewTabAccessibilityLabel';
+import Box from './Box';
+import VRButtonSocial from './ButtonSocial/VRButtonSocial';
+import { useDefaultLabelContext } from './contexts/DefaultLabelProvider';
+import Flex from './Flex';
+import Icon from './Icon';
+import InternalLink from './Link/InternalLink';
+import Text from './Text';
+import useInExperiment from './useInExperiment';
+
+const TEXT_OPTIONS = {
+  1: 'Login with',
+  2: 'Continue with',
+  3: 'Sign up with',
+};
+
+const SERVICES_OPTIONS = {
+  1: 'Apple',
+  2: 'Facebook',
+  3: 'Google',
+  4: 'Email',
+};
+
+type ButtonProps = {
+  /**
+   * Label to provide more context around ButtonLink’s function or purpose. See the [Accessibility guidelines](/foundations/accessibility) to learn more.,
+   */
+  accessibilityLabel?: string;
```

Comment:
I dont think we might need the accessibility label...

I think our default labels should be enough?